test/OpenSpaceToolkit/Astrodynamics/Trajectory/Propagator.test.cpp

```diff
@@ -64,6 +64,7 @@ using ostk::physics::Environment;
 using ostk::physics::coord::Frame;
 using ostk::physics::coord::Position;
 using ostk::physics::coord::Velocity;
+using ostk::physics::env::Object;
```

Comment:
Maybe I'm blind, but is this used anywhere?

Reply:
It's used on line 2043 for instantiating the Environment. in line 2052.